src/simmate/apps/materials_project/workflows/population_analysis/bader.py

```diff
@@ -37,18 +36,19 @@ def run_config(
             directory=prebader_dir,
         ).result()
 
-        # Setup chargecars for the bader analysis and wait until complete
-        chgcomb_dir = directory / PopulationAnalysis__Bader__CombineChgcars.name_full
-        PopulationAnalysis__Bader__CombineChgcars.run(
-            directory=chgcomb_dir,
-            previous_directory=prebader_dir,
-        ).result()
+        # # Setup chargecars for the bader analysis and wait until complete
+        # chgcomb_dir = directory / PopulationAnalysis__Bader__CombineChgcars.name_full
+        # PopulationAnalysis__Bader__CombineChgcars.run(
+        #     directory=chgcomb_dir,
+        #     previous_directory=prebader_dir,
+        # ).result()
 
         # And run the bader analysis on the resulting chg denisty
         bader_dir = directory / PopulationAnalysis__Bader__Bader.name_full
         PopulationAnalysis__Bader__Bader.run(
             directory=bader_dir,
-            previous_directory=chgcomb_dir,
+            # previous_directory=chgcomb_dir,
+            previous_directory=prebader_dir,
```

Comment:
go ahead and just remove the code you've commented out. If you wanted to keep it, make it a kwarg option that let's the user choose whether to call the combine chargecars exe -- that would mean this code goes where you have the "Grid.sum_grids" and you'd have a `use_henk_sun=False` or something like that as the default